### PR TITLE
feat(commands): quick reasons

### PR DESCRIPTION
**Pull request by JPBM135**
## What is this
Using the power of autocomplete interactions, this system allows reasons to be quickly utilized in moderation actions while still allowing mods to enter custom reasons if necessary!

> **Warning**
> This PR requires a command re-deploy

#### Reasons scraped from the `mod log` channel
- COPPA
- DM harassment
- Insults
- NSFW
- Off-topic in support
- Possible scam
- Rude on support
- Selfbot
- Shitpost
- Slurs
- Spam
- Suspicious or spam account
- TOS violation
- Troll

## Proof of concept
image
image

### CODE REVIEW

**Review by almostSouji — CHANGES_REQUESTED** [review]
~~Please add the raw input to the autocomplete response set to allow custom input as well~~
~~(remember that this needs to be truncated due to name length restrictions)~~
not needed

File: packages/yuudachi/locales/en-US/translation.json [hunk 1]
```diff
@@ -563,9 +579,17 @@
...
+				"fail_titles": [
+					"Member id",
+					"Username",
+					"Error"
+				]
 			}
 		}
 	}
-}
+}
```
Comment (almostSouji) [inline packages/yuudachi/locales/en-US/translation.json:595]: newline change

**Review by almostSouji — CHANGES_REQUESTED** [review]

**Comment by JPBM135** [comment]
> Please add the raw input to the autocomplete response set to allow custom input as well
> (remember that this needs to be truncated due to name length restrictions)

Good idea, although it's not necessary, it's a better UI experience

File: packages/yuudachi/locales/en-US/translation.json [hunk 2]
```diff
@@ -48,7 +48,23 @@
 					"no_reference_case": "Cannot find reference case `#{{case}}`",
 					"already_processed": "{{- user}} with case {{- case}} has no running restrictions.",
 					"no_reference_report": "Cannot find reference report `#{{report}}`"
-				}
+				},
+				"reasons": [
+					"Coppa",
```
Comment (appellation) [inline packages/yuudachi/locales/en-US/translation.json:53]: COPPA is an acronym, so this should probably be uppercased to be consistent with the other acronyms here.

**Review by appellation — COMMENTED** [review]
Can you remove the unrelated changes from this PR? I'm not sure why these `continue`s have been removed, but it doesn't seem to be related to this functionality.

**Comment by JPBM135** [comment]
> Can you remove the unrelated changes from this PR? I'm not sure why these `continue`s have been removed, but it doesn't seem to be related to this functionality.

Of course, I Will revert gettin home

File: packages/yuudachi/locales/en-US/translation.json (hunk 2 above)
Comment (JPBM135) [reply packages/yuudachi/locales/en-US/translation.json]: Fixed in 68523ac

**Comment by JPBM135** [comment]
The lint will fail bc of the solar js errors

File: packages/yuudachi/locales/en-US/translation.json [hunk 3]
```diff
@@ -725,4 +749,4 @@
 			}
 		}
 	}
-}
+}
```
Comment (iCrawl) [inline packages/yuudachi/locales/en-US/translation.json:752]:
```suggestion
}

```

**Review by iCrawl — APPROVED** [review]

**Pull request merged**